app/src/main/AndroidManifest.xml

```diff
@@ -28,7 +28,7 @@
         <meta-data android:name="com.google.android.gms.games.APP_ID"
             android:value="@string/app_id" />
         <meta-data android:name="com.google.android.gms.version"
-            android:value="@integer/google_play_services_version"/>
+            android:value="@integer/google_play_services_version" />
```

Comment:
revert

Reply:
But in all XML files we have spaces before closing tag

Reply:
Even in tag before this one you can see a space.